src/bids_validator/test_bids_validator.py

```diff
@@ -65,9 +65,14 @@ def test_datasets(validator, fname):
         ('/T1w.json', True),
         ('/dataset_description.json', True),
         ('/README', True),
+        ('/README.md', True),
+        ('/README.rst', True),
         ('/CHANGES', True),
         ('/participants.tsv', True),
+        ('/participants.json', True),
         ('/sub-01/anat/sub-01_T1w.nii.gz', False),
+        ('/RADME', False),  # typo
+        ('/CANGES', False),  # typo
     ],
 )
 def test_top_level(validator, fname, matches):
```

Comment:
In retrospect, I don't like parametrizing on `matches`. I would rather just split it into two functions, e.g.,

```python
@pytest.mark.parametrize('fname', valid_list)
def test_is_top_level(validator, fname):
    assert validator.is_top_level(fname)

@pytest.mark.parametrize('fname', invalid_list)
def test_is_not_top_level(validator, fname):
    assert not validator.is_top_level(fname)
```

If you've got the energy, would you do this? If not, I'll just create an issue, and might sooner or later get around to it.

Reply:
do you want to apply this to all tests?

Reply:
Yes.